project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject clj-commons/iapetos "0.1.10"
+(defproject org.clojars.nanit/iapetos "0.1.11"
```

Comment:
This doesn't seem quite right :)

Reply:
sorry I wasn't sure if this is going to get any attention so I released it to our private clojars repo
fixed by https://github.com/clj-commons/iapetos/pull/48/commits/9b1b0655af4dc493bdf4e29dd68a61cfc77bfd22